impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -38,6 +38,42 @@ define([
 
   return {
     rules: [
+      // line/barLine models
+      {
+        select: {
+          priority: -1,
+          type: [
+            "pentaho/visual/models/line",
+            "pentaho/visual/models/barLine"
+          ]
+        },
+        apply: {
+          props: {
+            // . line
+            lineWidth: {
+              value: 2
+            }
+          }
+        }
+      },
+
+      // heatGrid model
+      {
+        select: {
+          priority: -1,
+          type: [
+            "pentaho/visual/models/heatGrid"
+          ]
+        },
+        apply: {
+          props: {
+            colorSet: {
+              value: "blue"
```

Comment:
Shouldn't this `colorSet` default value  be accompanied with changes to the "blue" color scales? See: https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/pentaho/visual/color/paletteRegistry.js#L288